applications/games_menu/modules/menu.c

```diff
@@ -1406,7 +1422,7 @@ bool SavePresetGame(PresetStruct *preset)
 		uint32 cdda_mode = CDDA_MODE_DISABLED;
 
 		StopCDDA();
```

Comment:
I think there is no longer any need to stop CDDA here.
Also don't stop it when you go to settings.

Reply:
Yes, Iremember the last time you told me that, I noticed it as an improvement, I just haven't had time to do performance testing on it yet.